imod/mf6/multimodel/exchange_creator_unstructured.py

```diff
@@ -70,8 +83,30 @@ def _compute_geometric_information(self) -> pd.DataFrame:
         Vj = centroid_2 - edge_coordinates[:, 0]
```

Comment:
By changing the face1 and face2 on lines 65 and 68 you might flip around centroid_1 and centroid_2 which in turn have affect on the computed Vi and Vj and thus the cl1 and cl2 values. By flipping those values cl1 and cl2 might be flipped incorrectly

Reply:
indeed, they can be flipped. But in a good way: cl1 refers to the distance from the cell centre of the cell  with the lower partition number to the shared edge; cl2 to that of the cell centre with the higher partition number to that edge. Now it is computed like that.